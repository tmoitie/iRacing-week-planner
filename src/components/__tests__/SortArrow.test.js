import React from 'react';
import { render } from '@testing-library/react';
import { describe, test } from '@jest/globals';
import SortArrow from '../SortArrow';

describe('components/SortArrow', () => {
  test('renders down correctly', () => {
    const { container } = render(<SortArrow sort={{ order: 'desc', key: 'licence' }} />);

    expect(container).toMatchSnapshot();
  });

  test('renders up correctly', () => {
    const { container } = render(<SortArrow sort={{ order: 'asc', key: 'licence' }} />);

    expect(container).toMatchSnapshot();
  });
});
