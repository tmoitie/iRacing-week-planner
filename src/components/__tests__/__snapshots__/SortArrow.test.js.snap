// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/SortArrow renders down correctly 1`] = `
<div>
  <span
    class="glyphicon glyphicon-triangle-bottom"
  />
</div>
`;

exports[`components/SortArrow renders up correctly 1`] = `
<div>
  <span
    class="glyphicon glyphicon-triangle-top"
  />
</div>
`;
